dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+import "./index.less";
+import { useDispatch } from "react-redux";
+import {
+  TextContent,
+  TextVariants,
+  Text,
+  Button,
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  Pagination,
+} from "@patternfly/react-core";
```

Comment:
Unless there is a reason to the contrary, it would be good to maintain this list in sorted (alphabetical) order.  (And, presumably, line 14, as well.)

Reply:
Ordering will be done for all imports in a separate PR

Reply:
Subsetting work into separate PRs often makes sense. However, you *added* this list here, and deferring the sorting means creating project "technical debt" to deal with later when we have plenty of other things to do. It already seems clear that you're going to need another commit to make changes; it'd be great if you'd sort the lists and run Prettier and all that other minor cleanup at the same time.